src/Plaza/getPlazas.js

```diff
@@ -0,0 +1,16 @@
+export default async function getPlaza() {
+    //const apiURL = 'https://park-inn-ispp-be.herokuapp.com/plazas/all';
+    return await fetch('http://localhost:8080/plazas/all', {
+        method: 'GET',
+        credentials:'same-origin',
+        headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+        }
+    })
+        .then(res => res.json())
+        .then(plaza => console.log(plaza))
+        .catch(error => console.error("Error: ",error));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
        .catch((error) => console.error("Error: ",error));
```